test/components/AuthForm_test.js

```diff
@@ -254,7 +222,7 @@ describe("AuthForm component", () => {
         target: { value: "http://test.server/v1" },
       });
       expect(serverField.value).eql("http://test.server/v1");
-      expect(authTypeField.value).eql("anonymous");
+      expect(authTypeField.value).eql("openid-google");
```

Comment:
The test was sort-of broken before because of a quirk in how the component worked. In the old authForm it would default to anonymous auth after you changed servers, and an async event would search history (from localStorage) and update the value again if it found a match (through componentDidUpdate). But this test didn't go further to check the updated match was found later.

In the new code this happens in one step. If we have history for the new server we immediately default to the previous auth method. So I had to update the unit test to check the immediate value we're setting.